<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pony Sorter</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="consolebox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>781</width>
      <height>261</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="all_buttons" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>291</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>180</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>End time</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_verynoisy">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>150</y>
       <width>104</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Very noisy (6)</string>
     </property>
     <property name="shortcut">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_chooseunmix">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>70</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Select iZotope+Unmix (9)</string>
     </property>
     <property name="shortcut">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_next">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>10</y>
       <width>41</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_previous">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>41</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="moodbox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>221</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button_listenorig">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Listen to original (1)</string>
     </property>
     <property name="shortcut">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_listenizo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Listen to iZotope (2)</string>
     </property>
     <property name="shortcut">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="transcriptbox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>781</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_clean">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>150</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Clean (4)</string>
     </property>
     <property name="shortcut">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_listenunmix">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Listen to iZotope+Unmix (3)</string>
     </property>
     <property name="shortcut">
      <string>3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Primary mood</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_chooseorig">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Select original (7)</string>
     </property>
     <property name="shortcut">
      <string>7</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="startbox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>200</y>
       <width>121</width>
       <height>32</height>
      </rect>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QLineEdit" name="characterbox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>261</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Character</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="endbox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>200</y>
       <width>121</width>
       <height>32</height>
      </rect>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_noisy">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>150</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Noisy (5)</string>
     </property>
     <property name="shortcut">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>180</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Start time</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_chooseizo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Select iZotope (8)</string>
     </property>
     <property name="shortcut">
      <string>8</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Transcript</string>
     </property>
    </widget>
    <widget class="QLabel" name="secondarymoodlabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>230</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>none (secondary)</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLabels">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionLoad_episode"/>
    <addaction name="actionSave_changes"/>
    <addaction name="actionAdd_audio_path"/>
   </widget>
   <widget class="QMenu" name="menuSplit">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionSplit_by_pony"/>
    <addaction name="actionGenerate_combined_source"/>
    <addaction name="actionGenerate_Audacity_labels"/>
    <addaction name="actionGenerate_Audacity_labels_all_episodes"/>
   </widget>
   <addaction name="menuLabels"/>
   <addaction name="menuSplit"/>
  </widget>
  <action name="actionSplit_by_pony">
   <property name="text">
    <string>Split by pony (hardcoded to Twilight)</string>
   </property>
  </action>
  <action name="actionLoad_episode">
   <property name="text">
    <string>Load episode</string>
   </property>
  </action>
  <action name="actionSave_changes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save changes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionGenerate_combined_source">
   <property name="text">
    <string>Generate combined source (not implemented)</string>
   </property>
  </action>
  <action name="actionGenerate_Audacity_labels">
   <property name="text">
    <string>Generate Audacity labels (current episode)</string>
   </property>
  </action>
  <action name="actionAdd_audio_path">
   <property name="text">
    <string>Add audio path(s)</string>
   </property>
  </action>
  <action name="actionGenerate_Audacity_labels_all_episodes">
   <property name="text">
    <string>Generate Audacity labels (all episodes)</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>button_previous</tabstop>
  <tabstop>button_listenorig</tabstop>
  <tabstop>button_listenizo</tabstop>
  <tabstop>button_listenunmix</tabstop>
  <tabstop>button_chooseorig</tabstop>
  <tabstop>button_chooseizo</tabstop>
  <tabstop>button_chooseunmix</tabstop>
  <tabstop>button_next</tabstop>
  <tabstop>radio_clean</tabstop>
  <tabstop>radio_noisy</tabstop>
  <tabstop>radio_verynoisy</tabstop>
  <tabstop>characterbox</tabstop>
  <tabstop>moodbox</tabstop>
  <tabstop>startbox</tabstop>
  <tabstop>endbox</tabstop>
  <tabstop>transcriptbox</tabstop>
  <tabstop>consolebox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
